cli/package.json

```diff
@@ -11,20 +11,28 @@
   },
   "scripts": {
     "test": "npm run test-unit",
-    "test-unit": "bin-up mocha --reporter mocha-multi-reporters --reporter-options configFile=../mocha-reporter-config.json",
+    "test-unit": "npm run dtslint && bin-up mocha --reporter mocha-multi-reporters --reporter-options configFile=../mocha-reporter-config.json",
     "test-watch": "bin-up mocha --watch",
     "test-dependencies": "bin-up deps-ok && dependency-check . --no-dev",
     "test-debug": "node --inspect --debug-brk $(bin-up _mocha)",
     "lint": "bin-up eslint --fix *.js bin/* lib/*.js lib/**/*.js test/*.js test/**/*.js",
+    "dtslint": "dtslint types",
     "prebuild": "npm run test-dependencies && node ./scripts/start-build.js",
     "build": "node ./scripts/build.js",
     "prerelease": "npm run build",
     "release": "cd build && releaser --no-node --no-changelog",
     "size": "t=\"$(npm pack .)\"; wc -c \"${t}\"; tar tvf \"${t}\"; rm \"${t}\";"
   },
+  "types": "types",
   "dependencies": {
     "@cypress/listr-verbose-renderer": "0.4.1",
     "@cypress/xvfb": "1.0.4",
+    "@types/blob-util": "1.3.3",
```

Comment:
I'm pretty sure these have to be normal dependencies for downstream projects. I haven't actually deployed and linked as `devDependencies`, but from other testing the types will fail and fallback to `any` if the types can't be found.

Reply:
yeah, I think they will have to be included then.